tensorflow_lite_support/cc/task/vision/proto/BUILD

```diff
@@ -242,3 +242,52 @@ cc_library(
     hdrs = ["embeddings_proto_inc.h"],
     deps = [":embeddings_cc_proto"],
 )
+
+# LandmarkDetector protos
+
+proto_library(
+    name = "landmark_detector_options_proto",
+    srcs = ["landmark_detector_options.proto"],
+    deps = [
+        "//tensorflow_lite_support/cc/task/core/proto:base_options_proto",
+        "//tensorflow_lite_support/cc/task/core/proto:external_file_proto",
+        "@org_tensorflow//tensorflow/lite/experimental/acceleration/configuration:configuration_proto",
```

Comment:
external_file_proto and configuration_proto are not needed.

Reply:
ok